simulation_parameters/Constants.cs

```diff
@@ -309,6 +309,7 @@ public static class Constants
     public const int MAX_BACTERIAL_COLONY_SIZE = 6;
     public const int MIN_BACTERIAL_LINE_SIZE = 3;
     public const int MAX_BACTERIAL_LINE_SIZE = 7;
+    public const float BACTERIAL_COLONY_TIME_BETWEEN_SPAWN = 0.2f;
```

Comment:
There should be a different constant `SPAWN_SYSTEM_MAX_SPAWNS_PER_FRAME = 2` that is used.